README.md

```diff
@@ -1,5 +1,8 @@
 # TorchServe
 
+## Security Changes
+TorchServe now enforces token authorization and model mode control by default. Check the following documentation for more information: [Token Authorization](https://github.com/pytorch/serve/blob/master/docs/token_authorization_api.md), [Model mode control](https://github.com/pytorch/serve/blob/master/docs/model_control_mode.md)
```

Comment:
Can you please add what is model mode control ? and whats the usecase?

You can add it here at the top https://github.com/pytorch/serve/blob/master/docs/model_control_mode.md

Reply:
Model control mode is explained in the doc in multiple places including https://github.com/pytorch/serve/blob/master/docs/model_control_mode.md#model-control-mode-default and near the top in https://github.com/pytorch/serve/blob/master/docs/model_control_mode.md#two-ways-to-set-model-control

Reply:
Can change the wording to be little bit more descriptive

Reply:
I think we need to first mention what is model control mode? and why is it needed and when do we use which option.

The README mainly describes the various options but doesn't give an idea of what this feature is and what does it address

Reply:
I believe that it is better for the users to read about the feature in the full document and the README should inform users about the change. Similarly for token authorization the README informs users of the change and for full details they can read in the doc.

Reply:
I changed the naming to `Model API Control` which provides little more clarity then Model Mode Control.